src/connectors/youtube.ts

```diff
@@ -551,3 +560,127 @@ function isVideoCategoryAllowed() {
 		videoCategory === categoryUnknown
 	);
 }
+
+/**
+ * Check if a chapter name is likely AI-generated and doesn't contain track information.
+ * AI-generated chapters often contain generic descriptions that don't help with scrobbling.
+ */
+function isAiGeneratedChapter(chapterName: string): boolean {
```

Comment:
```suggestion
function isAiGeneratedChapter(chapterName: string | null): boolean {
```

Should be nullable as we pass in a nullable type is fine as we immediately check it